src/test/javascript/portal/cart/DownloadPanelSpec.js

```diff
@@ -234,6 +235,17 @@ describe("Portal.cart.DownloadPanel", function() {
         });
     });
 
+    describe('getEmbededTitle', function() {
+
+        it('Extracts the real title super trickerly embeded title in the description field from Geonetwork', function() {
```

Comment:
I'm not really sure what's going on here - are there metadata records which have a `description` field, which in turn contain a "title", given in parentheses?  Is that right?  (Example metadata record, please).

This seems a bit weird to me - why not put the "title" separately in the metadata record?